fmriprep/cli/run.py

```diff
@@ -573,7 +573,19 @@ def build_workflow(opts, retval):
       * Run identifier: {uuid}.
     """.format
 
-    output_spaces = opts.output_space or []
+    # Reduce to unique space identifiers
+    output_spaces = sorted(set(opts.output_space))
+
+    # If FS is not run, drop all fs* output spaces
+    if not opts.run_reconall:
+        output_spaces = [item for item in output_spaces if not item.startswith('fs')]
+
+    # Map fsaverage to fsaverage6
+    if 'fsaverage' in output_spaces:
+        logger.warning('Selected "fsaverage" output space has been mapped to "fsaverage6"')
```

Comment:
I don't think we should override user selected output spaces.

Reply:
fsaverage is an alias for fsaverage6, isn't that correct?

Reply:
The reason for this is that our cifti interface fails if it is passed "fsaverage". We could do the mapping silently as FS does, but I prefer being explicit.

Reply:
No. fsaverage6 is a deformed order 6 icosahedron. fsaverage is order 7.

Reply:
The reason we require fsaverage6 for cifti is because fsaverage would be huge.

Reply:
Gotcha